lib/Site.js

```diff
@@ -209,7 +209,7 @@ Site.prototype.buildSourceFiles = function () {
 };
 
 /**
- * Rebuild source files that are affected by change in filePath
+ * Rebuild pages that are affected by change in filePath
```

Comment:
I've changed this to make it consistent with https://github.com/MarkBind/markbind-cli/pull/17/commits/831637cbc93499bb54dbf333e84df866fb479661#diff-e5f3ecb7effaa21db6153e4647ea22bcR197

Rationale is that the method has no knowledge of what is being rebuilt besides the pages. Also should we start documenting promises?